pkg/handler/eval_cache.go

```diff
@@ -59,6 +63,39 @@ func (ec *EvalCache) Start() {
 	}()
 }
 
+func (ec *EvalCache) GetByTags(tags []string) []*entity.Flag {
+	ec.mapCacheLock.RLock()
+	defer ec.mapCacheLock.RUnlock()
+
+	results := []*entity.Flag{}
+	for _, t := range tags {
+		s := util.SafeString(t)
```

Comment:
t is already a string type, so no need to cast to string here

Reply:
Perfect, we'll remove this 👍